repos/system_upgrade/el7toel8/actors/addupgradebootentry/tests/unit_test.py

```diff
@@ -0,0 +1,23 @@
+from leapp.libraries.actor import library
+from leapp.libraries import stdlib
+
+
+class call_mocked(object):
+    def __call__(self, args, split):
+        self.args = args
+
+
+def test_add_boot_entry(monkeypatch):
+    monkeypatch.setenv('LEAPP_DEBUG', '1')
+    monkeypatch.setattr(stdlib, 'call', call_mocked())
+
+    def get_boot_file_paths_mocked():
+        return '/abc', '/def'
+    monkeypatch.setattr(library, 'get_boot_file_paths', get_boot_file_paths_mocked())
+
+    library.add_boot_entry()
+
+    assert library.get_boot_file_paths.args.join(' ') == ('/usr/sbin/grubby --add-kernel=/abc --initrd=/def'
```

Comment:
wouldn't that be rather  `'  '.join(library.get_boot_file_paths.args) == ...`

Reply:
Indeed. This was like "pushing whatever I had before calling it a day" without really running the tests locally. I guess the thing is that I switch between the branches all the time causing conflicts when I have some stuff drafted and git stash is just a partial solution. But I'll try to run the tests before next pushes.